core/lib/list_item_hunter.js

```diff
@@ -13,6 +13,7 @@
 var list_item_hunter = function () {
 
   var extend = require('util')._extend,
+    JSON = require('json5'),
```

Comment:
I wonder if it might be more appropriate to name this `JSON5`, so that people don't mistake it for the built-in `JSON` object.